test/hw3/test_task3.py

```diff
@@ -0,0 +1,36 @@
+from All_home_works.hw3.task3 import Filter, make_filter, sample_data
+
+
+def test_filter():
+    expected = list(range(2, 100, 2))
```

Comment:
Why do you need to have 50 elements for tests?

Reply:
Fixed